package.json

```diff
@@ -35,7 +35,7 @@
   ],
   "dependencies": {
     "@observablehq/inspector": "^3.0.0",
-    "@observablehq/stdlib": "^3.0.0",
+    "@observablehq/stdlib": "https://github.com/observablehq/stdlib#44e7a2e",
```

Comment:
Is this temporary until publishing a new version of stdlib?

Reply:
Yep, we'll have a sort of 'wave' of publishes once this project as a whole works. This middleground is so that it's possible to test the application as a whole.